tests/local_client_server_tcp_e2e/local_client_server_tcp_e2e.c

```diff
@@ -26,6 +26,23 @@ static TEST_MUTEX_HANDLE g_dllByDll;
 
 #define TEST_TIMEOUT 30 // seconds
 
+static int generate_port_number()
+{
+	int port_number;
+	const char* comp_time = __TIME__;
+	int comp_secs = 10 * (comp_time[6] - '0') + (comp_time[7] - '0');
+	int comp_mins = 10 * (comp_time[3] - '0') + (comp_time[4] - '0');
+	int comp_hours = 10 * (comp_time[0] - '0') + (comp_time[1] - '0');
```

Comment:
Actually I believe this statement is not true. The unit is only compiled once looking at Jenkins logs.

---
In reply to: [109773214](https://github.com/Azure/azure-uamqp-c/pull/74#discussion_r109773214) [](ancestors = 109773214)